sentry_streams/src/routers.rs

```diff
@@ -0,0 +1,106 @@
+use crate::callers::call_python_function;
+use crate::routes::{Route, RoutedValue};
+use pyo3::prelude::*;
+use sentry_arroyo::processing::strategies::run_task::RunTask;
+use sentry_arroyo::processing::strategies::{InvalidMessage, ProcessingStrategy, SubmitError};
+use sentry_arroyo::types::{InnerMessage, Message};
+
+fn route_message(
+    route: &Route,
+    callable: &Py<PyAny>,
+    message: Message<RoutedValue>,
+) -> Result<Message<RoutedValue>, SubmitError<RoutedValue>> {
+    if message.payload().route != *route {
+        return Ok(message);
+    }
+    let dest_route = call_python_function(callable, &message);
+    Python::with_gil(|py| match dest_route {
```

Comment:
Is it possible to move the Python call out of this main loop? This is more of a stylistic thing:

```
match dest_route {
    Ok(dest_route) => {
        let new_waypoint = extra_python_string(dest_route);
        ...
}
```

Reply:
You are right, it should work.

Reply:
done